components/google-cloud/setup.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2021 Google LLC. All Rights Reserved.
```

Comment:
This file does not seem to be related to the components that are alrady in the goodle-cloud directory. Could we move it to a grouping subdirectory?

Reply:
the long term goal for the skd is to cover all components ( including bigquery, dataflow , etc) under this folder not just limited to aiplatfrom training, with that would the current location be ok or should we move everything down one level to something like ( google-cloud/src/python/...)

Reply:
Done. Thank you for discussion and guidance on this, based on the discussion I updated the structure.